kip-0013.md

```diff
@@ -7,10 +7,14 @@
   Status: draft
 ```
 
-We propose tracking the transient storage consumption of transactions as a `mass` and bounding such consumption to a reasonable size for node operators.
+We propose tracking the transient storage consumption of transactions as a `mass` and bounding such consumption to a reasonable size for node operators. In the Kaspa jargon, the word `mass`` is used for sizes that are used to limit transaction throughput due to their externality.
 
 # Motivation
 
+Since KIP9, transactions now consume two types of masses, compute mass and storage mass. These masses are used to limit some consumption of resource per block (compute and persitent storage, respectively). However, either of these directly limit the block size, byte-wise. We need to have a way to limit block byte sizes directly to have finer control over the worst-case storage used by a node.
```

Comment:
"However, either of these". Did you mean "neither"?

Reply:
"We need to have a way to limit block byte sizes directly to have finer control over the worst-case storage used by a node." Maybe also mention bandwidth?